elements/pl-file-editor/pl-file-editor.js

```diff
@@ -174,26 +209,58 @@ window.PLFileEditor.prototype.initSettingsButton = function (uuid) {
       var fontSize = $(this).val();
       that.editor.setFontSize(fontSize);
     });
+    that.modal.find('#modal-' + uuid + '-keyboardHandler').change(function () {
+      var keyBinds = $(this).val();
+      if (keyBinds !== 'ace/keyboard/default') {
+        that.editor.setKeyboardHandler(keyBinds);
+      } else {
+        that.editor.setKeyboardHandler(null);
+      }
+    });
```

Comment:
Hey Nathan, should we keep the keybinding preview? I felt like having a preview of the keybinding's cursor is fun (because they varied in colors), but apart from this, users cannot try those keybinds, unless they quit the modal.

Reply:
IMO it would be fine to remove the preview.